system-tests/tests/resources/jwt/identity-hub-owner/private-key.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIMH5nIIP0d/AI1D1L6DpIscPL3EX5x5XHxrBnt2+TduzoAoGCCqGSM49
```

Comment:
why are these two keys the same? this is confusing and may lead to wrong test implementation

Reply:
Good point, I have created separate keys. We have the same situation in the MVD code, perhaps we should consider having different keys for each participant there as well.